src/main/java/me/zeroeightsix/kami/mixin/client/player/MixinEntityPlayerSP.java

```diff
@@ -65,6 +68,17 @@ public void closeScreen(Minecraft minecraft, GuiScreen screen) {
         if (PortalChat.INSTANCE.isDisabled()) Wrapper.getMinecraft().displayGuiScreen(screen);
     }
 
+    @Override
+    public void travel(float strafe, float vertical, float forward) {
+        PlayerTravelEvent event = new PlayerTravelEvent();
+        KamiEventBus.INSTANCE.post(event);
+        if (event.getCancelled()) {
+            move(MoverType.SELF, motionX, motionY, motionZ);
+        } else {
+            super.travel(strafe, vertical, forward);
+        }
+    }
+
```

Comment:
Please never use override or overwrite in mixin, it breaks compatibility with other mods. Use Inject instead

Reply:
Inject will only work if we inject into the MixinEntityPlayer, I don't have a good way to detect if we are on the client side yet. The root cause of the problem is calling an event within the EntityPlayerMP class inheritance.

Any ideas?

Reply:
hope nobody else tries this fix and just yolo it

I don't think anybody else uses the profiler